### DISCUSSION TITLE
社区运营 Orbit 工具推荐

### DISCUSSION THREAD

**Discussion by will-ww** [kaiyuanshe/ONES#78] [案例分享 & 最佳实践]
在社区运营以及 DevRel 这块，https://orbit.love/ 做的挺好，值得借鉴。

![image](https://user-images.githubusercontent.com/15010826/167276773-3dba9ca6-57c5-4567-81a0-4d24650b1c65.png)

![image](https://user-images.githubusercontent.com/15010826/167276804-e8dbc1db-484f-4864-bb72-2412fc30185e.png)

![image](https://user-images.githubusercontent.com/15010826/167276819-360c64a1-a2fe-475f-9f29-032f051ecb3d.png)

Orbit 的创始人的访谈：https://www.youtube.com/watch?v=j1s7t1riQE4

**Comment by LinuxSuRen** [comment]
这个工具确实很酷！赞👍！

**Comment by wangleo61** [comment]
哈哈，这个不错呀

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: Will ***@***.***>
发送时间: Sunday, May 8, 2022 8:38 AM
收件人: ***@***.***>
抄送: ***@***.***>
主题: [kaiyuanshe/ONES] 社区运营 Orbit 工具推荐 (Discussion #78)


在社区运营以及 DevRel 这块，https://orbit.love/ 做的挺好，值得借鉴。

[image]<https://user-images.githubusercontent.com/15010826/167276773-3dba9ca6-57c5-4567-81a0-4d24650b1c65.png>

[image]<https://user-images.githubusercontent.com/15010826/167276804-e8dbc1db-484f-4864-bb72-2412fc30185e.png>

[image]<https://user-images.githubusercontent.com/15010826/167276819-360c64a1-a2fe-475f-9f29-032f051ecb3d.png>

Orbit 的创始人的访谈：https://www.youtube.com/watch?v=j1s7t1riQE4

―
